score-xml/src/main/java/io/cloudslang/content/xml/actions/AddAttribute.java

```diff
@@ -28,21 +31,23 @@
      */
     @Action(name = "Add Attribute",
             outputs = {
-                    @Output(Constants.OutputNames.RESULT_TEXT),
-                    @Output(Constants.OutputNames.RETURN_RESULT),
-                    @Output(Constants.OutputNames.RESULT_XML)},
+                    @Output(Constants.Outputs.RESULT_TEXT),
```

Comment:
We should clarify if we go forward with RESULT_TEXT or RETURN_CODE output. The existing operations in score-ations use the returnCode output to establish the success or failure status, with no resultText. If we proceed with this behaviour there is inconsistency between operations.